app/bundles/LeadBundle/Model/DoNotContact.php

```diff
@@ -144,7 +154,7 @@ public function isContactable(Lead $contact, $channel)
             $channel = key($channel);
         }
 
-        /** @var \Mautic\LeadBundle\Entity\DoNotContact[] $entries */
+        /** @var DNC[] $entries */
```

Comment:
```suggestion
        /** @var Collection<int, DNC> $dncEntries */
```

Reply:
@escopecz  Removed the annotation, as it's available in the method, which is called.